explore/src/main/scala/explore/config/AdvancedConfigurationPanel.scala

```diff
@@ -114,121 +137,295 @@ sealed abstract class AdvancedConfigurationPanelBuilder[
+
+  private def secondsToDuration(secs: NonNegInt): NonNegDuration =
+    NonNegDuration.unsafeFrom(secs.value.toLong.seconds)
+
+  private def optExposureTimeModeToExpTimeSecs(oetm: Option[ExposureTimeMode]): Option[NonNegInt] =
+    oetm.flatMap(etm => ExposureTimeMode.exposureTime.getOption(etm)).map(durationToSeconds)
+
+  val itcNoneMsg = "No ITC Results"
+
   val component =
     ScalaFnComponent
       .withHooks[Props]
-      .render { props =>
+      .useStateViewBy { props =>
+        implicit val ctx = props.ctx
+        overrideExposureTimeMode(props.scienceModeAdvanced).get
+          .map(ExposureTimeModeType.fromExposureTimeMode)
+      }
+      .useStateViewBy { (props, _) =>
+        implicit val ctx = props.ctx
+        val oetm         = overrideExposureTimeMode(props.scienceModeAdvanced).get
+        optExposureTimeModeToExpTimeSecs(oetm)
+      }
+      .useEffectWithDepsBy { (props, _, _) =>
         implicit val ctx = props.ctx
+        optExposureTimeModeToExpTimeSecs(overrideExposureTimeMode(props.scienceModeAdvanced).get)
+      }((_, _, expTimeOverride) =>
+        newExpTime =>
+          if (newExpTime === expTimeOverride.get) Callback.empty
+          else expTimeOverride.set(newExpTime)
+      )
+      .render {
+        (
+          props,
+          exposureModeEnum,
+          expTimeOverrideSecs
+        ) =>
+          implicit val ctx = props.ctx
+
+          val exposureModeView = overrideExposureTimeMode(props.scienceModeAdvanced)
+
+          val exposureCountView: Option[View[NonNegInt]] =
+            exposureModeView
+              .mapValue((v: View[ExposureTimeMode]) =>
+                v.zoom(ExposureTimeMode.exposureCount).asView
+              )
+              .flatten
 
-        def dithersControl(onChange: Callback): VdomElement =
-          ReactFragment(
-            <.label("λ Dithers", HelpIcon("configuration/lambda-dithers.md")),
-            InputWithUnits(
-              id = "dithers",
-              value = explicitWavelengthDithers(props.scienceModeAdvanced).withOnMod(_ => onChange),
-              validFormat = ExploreModelValidators.dithersValidSplitEpi,
-              changeAuditor =
-                ChangeAuditor.bigDecimal(integers = 3, decimals = 1).toSequence().optional,
-              units = "nm"
+          val exposureTimeView: Option[View[NonNegDuration]] =
+            exposureModeView
+              .mapValue((v: View[ExposureTimeMode]) => v.zoom(ExposureTimeMode.exposureTime).asView)
+              .flatten
+
+          val signalToNoiseView: Option[View[PosBigDecimal]] =
+            exposureModeView
+              .mapValue((v: View[ExposureTimeMode]) =>
+                v.zoom(ExposureTimeMode.signalToNoiseValue).asView
+              )
+              .flatten
+
+          def dithersControl(onChange: Callback): VdomElement =
+            ReactFragment(
+              <.label("λ Dithers", HelpIcon("configuration/lambda-dithers.md")),
```

Comment:
Maybe I can read the help to learn what a dither is

Reply:
Or ask Raul. He knows. 😄